frontend/packages/console-app/src/actions/creators/deployment-factory.ts

```diff
@@ -81,7 +86,7 @@ export const DeploymentActionFactory: ResourceActionFactory = {
     id: `edit-deployment`,
     label: i18next.t('console-app~Edit {{kind}}', { kind: kind.kind }),
     cta: {
-      href: `/edit-deployment/ns/${obj.metadata.namespace}?name=${obj.metadata.name}&kind=${kind.kind}`,
+      href: `${resourceObjPath(obj, kind.crd ? referenceForModel(kind) : kind.kind)}/form`,
```

Comment:
@debsmita1 So the previous URL had name and kind as query-params. But the new one will not have them in the same form. We should make sure that it doesnt break the code because these might be getting used in form.